### DISCUSSION TITLE
I need some advice on HDRI texture usage, gamma, color transform

### DISCUSSION THREAD

**Discussion by orarugogerincufelpattano** [mitsuba-renderer/mitsuba-blender#95] [General]
Hi! I'm struggling to create images when using HDRIs. 
Depending on the HDRI used, the reflections of the environment are very over saturated. The HDRI itself (if viewed in HDRIShop or Gimp) doesn't have that over saturated look. 
Viewing an HDRI in a software like Gimp on a monitor... there are already a lot of color transformation applied.
The only parameter in Mitsuba I see, is the Scale value to modify the exposure of the HDRI.
Is there any other way to tweak the HDRI at rendering. For example setting a different Gamma value?
Or do I have to apply some View Transform (like in Blender with Filmic/AgX) to the final image?
Can someone advice on how to use HDRIs properly in Mitsuba? Thanks.

**Comment by orarugogerincufelpattano** [comment]
Damn, it is my image viewer. I use Kobe ImageGlass as a viewer and apparently it has some problems displaying EXR files properly.
When I use Gimp to display the final rendered images, they are looking fine. 
Sorry for the confusion.
I leave the question still open, if there is any advice or best practice even if there is actually no problem.

**Comment by bathal1** [comment]
Hi,

Mitsuba typically renders EXr images, so you need to use a suitable image viewer to visualize the renderings. We recommend using [tev](https://github.com/Tom94/tev) for that purpose. 

Regarding using HDRIs in Mitsuba, it should be as trivial as doing it in Blender.

I will close this as it is unrelated to the exporter.
